packages/react-components/src/components/Button/Button.module.scss

```diff
@@ -28,7 +28,7 @@ $base-class: 'btn';
   }
 
   &:focus {
-    box-shadow: 0 0 1px 2px rgb(var(--color-action-default-rgb) 0.7);
+    filter: drop-shadow(0 0 1px 2px rgb(var(--color-action-default-rgb) 0.7));
```

Comment:
It will be nice if You add "before" and "after" images to the description because now we can't easily verify this change without playing with code in the inspector. WDYT?

Reply:
Yes, master!